internal/metrics/identity.go

```diff
@@ -0,0 +1,77 @@
+// Copyright © 2025 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var NodeIdentityDXCertMismatchGauge *prometheus.GaugeVec
+var NodeIdentityDXCertExpiryGauge *prometheus.GaugeVec
+
+const (
+	NodeIdentityDXCertMismatch = "ff_multiparty_node_identity_dx_mismatch"
+	NodeIdentityDXCertExpiry   = "ff_multiparty_node_identity_dx_expiry_epoch"
+)
+
+func InitIdentityMetrics() {
+	NodeIdentityDXCertMismatchGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: NodeIdentityDXCertMismatch,
+		Help: "Status of node identity DX cert mismatch",
+	}, namespaceLabels)
+
+	NodeIdentityDXCertExpiryGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: NodeIdentityDXCertExpiry,
+		Help: "Timestamp, in Unix epoch format, of node identity DX cert's expiry",
+	}, namespaceLabels)
+}
+
+func RegisterIdentityMetrics() {
+	registry.MustRegister(NodeIdentityDXCertMismatchGauge)
+	registry.MustRegister(NodeIdentityDXCertExpiryGauge)
+}
+
+// TODO should this type live elsewhere ??
```

Comment:
I wonder if this should be exposed in the `/status/multiparty` API we recently added https://hyperledger.github.io/firefly/latest/swagger/#/Default%20Namespace/getStatusMultiparty